### ISSUE TITLE
Vertical video image scaled disproportionately

### ISSUE THREAD

**Issue by pbdco** [bigbluebutton/bigbluebutton#9653]
**Describe the bug**
The video image is deformed on playback if participant use camera vertically during a meeting (example: mobile users).

**To Reproduce**
1. Access to meeting from mobile
2. Record
3. See playback

**Expected behavior**
Playback with verticual video same as recorded / or image scaled proportionally (doesn't maintain aspect ratio)

**Actual behavior**
The "vertical" recorded video images are shrinked on height and stretched on width completely distorting the image.

**Screenshots**
Example:

Original: https://imgur.com/a/To1DEER
On playback: https://imgur.com/a/MbKJlgI

**Desktop (please complete the following information):**
 - OS: Mac os latest
 - Browser: chrome
 - Version: latest

**Smartphone (please complete the following information):**
 - Device: recorded on iPhone 11
 - OS: latest
 - Browser: safari
 - Version: latest

**Additional context**

**Comment by tglemser** [comment]
Maybe this is a related problem: If the mobile user holds his device at the beginning with aspect ratio 3:4 and then changes the horizont to 4:3, the changes of the aspect ratio are not reflected in the recording after processing.

The RAW File at the beginning is 480x640px and then switches correctly to 640x480. 

The recording after processing is constantly 640x480px. 

The 480x640px part is scaled correctly with two white borders on the left and the right. The 640x480 part is scaled disproportionately in the same area as the 480x640px part.

**Comment by pbdco** [comment]
> Maybe this is a related problem: If the mobile user holds his device at the beginning with aspect ratio 3:4 and then changes the horizont to 4:3, the changes of the aspect ratio are not reflected in the recording after processing.
> 
> The RAW File at the beginning is 480x640px and then switches correctly to 640x480.
> 
> The recording after processing is constantly 640x480px.
> 
> The 480x640px part is scaled correctly with two white borders on the left and the right. The 640x480 part is scaled disproportionately in the same area as the 480x640px part.

Exactly

**Comment by pedrobmarin** [comment]
@prlanzarin could this still be an issue?

**Comment by prlanzarin** [comment]
Yes, this is still a thing. Not only for webcams, but screen sharing as well.

The post-processed videos (camera, screen) are getting bound to an initial aspect ratio. Any intermediate resizes (ie a smartphone camera flip, a shared application resize) that change the aspect ratio significantly might get the stream distorted.

**Comment by FreeScholar** [comment]
*****Solution:** 
Do not record in vertical position. Ever.
Perhaps include a small text warning when mobile cam is enabled. ( *Hold phone horizontally - or Record in horizontal position )
Vertical recordings are just bad.
